devfiles/java-web-spring/devfile.yaml

```diff
@@ -12,6 +12,7 @@ components:
   -
     type: chePlugin
     id: redhat/java/latest
+    memoryLimit: 1512Mi
```

Comment:
This *could* potentially be dropped, as the default used for `redhat/java/latest` is `1500Mi`. However this would mean any increases to the plugin-registry default would break this devfile again.